### PR TITLE
Implement first version of ecdsa 2019 cryptosuite.

### PR DESCRIPTION
**Pull request by kezike**
This PR implements the first version of the ECDSA 2019 crypto suite.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,13 +14,12 @@
     "lib/**/*.js"
   ],
   "dependencies": {
-    "@digitalbazaar/ecdsa-multikey": "^1.0.0",
+    "@digitalbazaar/ecdsa-multikey": "github:digitalbazaar/ecdsa-multikey#fix-sign-and-verify-issues",
```
Comment (kezike) [inline package.json]: TODO: Update this line after this branch has been deployed to the npm registry.

File: test/Ecdsa2019Cryptosuite.spec.js [hunk 2]
```diff
@@ -6,6 +6,21 @@ import {expect} from 'chai';
 import jsigs from 'jsonld-signatures';
 const {purposes: {AssertionProofPurpose}} = jsigs;
 
+import * as EcdsaMultikey from '@digitalbazaar/ecdsa-multikey';
+import {
+  credential,
+  ecdsaMultikeyKeyPair,
+  ecdsaSecp256KeyPair,
+  ecdsaSecp384KeyPair,
+  ecdsaSecp521KeyPair
```
Comment (dlongley) [inline test/Ecdsa2019Cryptosuite.spec.js]: So this cryptosuite is only intended to support secp256r1, none of the others -- so we should remove those ... unless there's something I'm missing.

**Review by dlongley — CHANGES_REQUESTED** [review]

File: test/Ecdsa2019Cryptosuite.spec.js (hunk 2 above)
Comment (kezike) [reply test/Ecdsa2019Cryptosuite.spec.js]: Yes, you're right. We wouldn't ever use these keys in this crypto suite. I have removed them.

File: test/Ecdsa2019Cryptosuite.spec.js [hunk 3]
```diff
@@ -19,51 +32,312 @@ describe('Ecdsa2019Cryptosuite', () => {
...
+          });
+        } catch(e) {
+          error = e;
+        }
+
+        expect(error).to.not.exist;
+        expect(verifier).to.exist;
+        verifier.algorithm.should.equal('P-256');
+        verifier.id.should.equal('https://example.edu/issuers/565049#zDnaekG' +
+        'ZTbQBerwcehBSXLqAg6s55hVEBms1zFy89VHXtJSa9');
```
Comment (dlongley) [inline test/Ecdsa2019Cryptosuite.spec.js]:
```suggestion
          'ZTbQBerwcehBSXLqAg6s55hVEBms1zFy89VHXtJSa9');
```

**Review by dlongley — APPROVED** [review]
Thanks! Just a tiny style nit and I think we're good here?

@gannan08 can you also take a look here?

File: package.json (hunk 1 above)
Comment (gannan08) [reply package.json]: This can be updated now.

**Review by gannan08 — APPROVED** [review]

**Comment by dlongley** [comment]
I plan on merging and releasing this later today unless I hear that we're still waiting on something.

**Comment by gannan08** [comment]
GTG. Not waiting on anything.

**Pull request merged**